seatunnel-common/src/main/java/org/apache/seatunnel/common/config/EnvConfigParserUtil.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.common.config;
+
+import org.apache.seatunnel.common.utils.FileUtils;
+
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EnvConfigParserUtil {
+
+    public static Map<String, String> parseEnvConfig() {
+        Path path = Common.configDir();
+        String os = System.getProperty("os.name").toLowerCase();
+
+        if (os.startsWith("win")) {
```

Comment:
This is not stable way to get env value. Maybe we can add all system enviorment variable and `seatunnel-env.sh` varibale into env value map.
Just execute `seatunnel-env.sh` before start jvm program can achieve it.